keyboards/redragon/k667/k667.c

```diff
@@ -0,0 +1,112 @@
+/* Copyright 2023 temp4gh
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "quantum.h"
+
+#ifdef RGB_MATRIX_ENABLE
+const is31_led PROGMEM g_is31_leds[RGB_MATRIX_LED_COUNT] = {     
```

Comment:
Trailing whitespace should be removed.

Reply:
Trailing whitespace removed.

Reply:
Both of these are still outstanding.

Reply:
Please check the commits: 03f69f4,thanx!

Reply:
![image](https://github.com/qmk/qmk_firmware/assets/4781841/e1e94bc7-e270-4ea2-9a1a-5894db325f5e)

Reply:
Got it ,I replaced contents of the json file with the results from format-json.Thanx!